src/ray/raylet/main.cc

```diff
@@ -38,6 +33,12 @@ int main(int argc, char *argv[]) {
   const std::string redis_password = (argc >= 15 ? std::string(argv[14]) : "");
   const std::string temp_dir = (argc >= 16 ? std::string(argv[15]) : "/tmp/ray");
 
+  InitShutdownRAII ray_log_shutdown_raii(ray::RayLog::StartRayLog,
+                                         ray::RayLog::ShutDownRayLog, argv[0],
+                                         ray::RayLogLevel::INFO,
+                                         /*log_dir=*/temp_dir + "/logs");
```

Comment:
Currently, all the log will be output to stderr and the python code (services.py) will redirect the logs to ‘/tmp/ray“. If you specify this parameter with non-empty string, there will be duplicated logs. Do you really mean this? Is it better not to provide the path if user does not provide this.

Reply:
Maybe you want to separate the Logging level for stderr and file?

Reply:
Even if not providing the `log_dir` path, Raylogs are still stored duplicated in /tmp and /tmp/ray/. 
And if `log_dir` path is provided, Raylogs are stored only with the level `severity_threshold`, I guess the original intension would be storing the logs with level higher than `severity_threshold`.
I modified the code again to fix the above questions. 
Look forward to your review.